### YamlMime:ManagedReference
items:
- uid: System.Runtime.Remoting.RemotingException
  id: RemotingException
  children:
  - System.Runtime.Remoting.RemotingException.#ctor
  - System.Runtime.Remoting.RemotingException.#ctor(System.String)
  - System.Runtime.Remoting.RemotingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Runtime.Remoting.RemotingException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RemotingException
  nameWithType: RemotingException
  fullName: System.Runtime.Remoting.RemotingException
  type: Class
  summary: "De uitzondering die wordt gegenereerd wanneer er iets een opgetreden tijdens het externe toegang fout is."
  remarks: "RemotingException maakt gebruik van de HRESULT COR_E_REMOTING, dat de waarde 0x8013150B heeft.       RemotingException gebruikt de standaardinstallatielocatie <xref:System.Object.Equals%2A>implementatie, die ondersteuning biedt voor verwijzingsgelijkheid.</xref:System.Object.Equals%2A>       Zie voor een lijst met eerste eigenschapswaarden voor een exemplaar van RemotingException, de RemotingException constructors."
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class RemotingException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor
  id: '#ctor'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException()
  nameWithType: RemotingException.RemotingException()
  fullName: System.Runtime.Remoting.RemotingException.RemotingException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.Runtime.Remoting.RemotingException&quot;> </xref> klasse met de standaardeigenschappen."
  remarks: "<xref:System.Runtime.Remoting.RemotingException>neemt over van de <xref:System.SystemException>klasse.</xref:System.SystemException></xref:System.Runtime.Remoting.RemotingException> De volgende tabel toont de eerste waarden voor een exemplaar van de klasse <xref:System.Runtime.Remoting.RemotingException>: | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`|   | <xref:System.Exception.Message%2A>| De lege tekenreeks (&quot;&quot;) |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.RemotingException>"
  syntax:
    content: public RemotingException ();
    parameters: []
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException(String)
  nameWithType: RemotingException.RemotingException(String)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.Runtime.Remoting.RemotingException&quot;> </xref> klasse met een opgegeven bericht."
  remarks: "<xref:System.Runtime.Remoting.RemotingException>neemt over van de <xref:System.SystemException>klasse.</xref:System.SystemException></xref:System.Runtime.Remoting.RemotingException> De volgende tabel toont de eerste waarden voor een exemplaar van de klasse <xref:System.Runtime.Remoting.RemotingException>: | Eigenschap van het Type | Voorwaarde |   |-------------------|---------------|   | <xref:System.Exception.InnerException%2A>| `null`|   | <xref:System.Exception.Message%2A>| De `message` tekenreeks |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.RemotingException>"
  syntax:
    content: public RemotingException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht waarin wordt uitgelegd waarom de uitzondering is opgetreden."
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException(SerializationInfo,StreamingContext)
  nameWithType: RemotingException.RemotingException(SerializationInfo,StreamingContext)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.Runtime.Remoting.RemotingException&quot;> </xref> klasse vanuit een geserialiseerde gegevens."
  remarks: "Deze constructor is aangeroepen tijdens de deserialisatie op het object exception verzonden via een stroom opnieuw samenstellen. (Zie voor meer informatie over serialisatie [XML en SOAP-serialisatie](~/add/includes/ajax-current-ext-md.md).)"
  syntax:
    content: protected RemotingException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Het object dat de gegevens geserialiseerd object bevat."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "De contextuele informatie over de bron- of doelserver van de uitzondering."
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions:
  - type: System.ArgumentNullException
    commentId: T:System.ArgumentNullException
    description: "The <code>info</code> parameter is <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>."
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException(String,Exception)
  nameWithType: RemotingException.RemotingException(String,Exception)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.Runtime.Remoting.RemotingException&quot;> </xref> klasse met een opgegeven foutbericht en een verwijzing naar de interne uitzondering dat de oorzaak van deze uitzondering."
  remarks: "Een uitzondering die is gegenereerd als direct resultaat van een eerdere uitzondering bevat een verwijzing naar de vorige uitzondering in de <xref:System.Exception.InnerException%2A>eigenschap.</xref:System.Exception.InnerException%2A> De <xref:System.Exception.InnerException%2A>eigenschap dezelfde waarde die is doorgegeven aan de constructor of `null` als de <xref:System.Exception.InnerException%2A>eigenschap levert niet de waarde van de binnenste uitzondering voor de constructor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       De volgende tabel toont de eerste waarden voor een exemplaar van de klasse <xref:System.Runtime.Remoting.RemotingException>: | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| De verwijzing naar de interne uitzondering. |   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.RemotingException>"
  syntax:
    content: public RemotingException (string message, Exception InnerException);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht waarin wordt uitgelegd waarom de uitzondering is opgetreden."
    - id: InnerException
      type: System.Exception
      description: "De uitzondering die de oorzaak van de huidige uitzondering. Als de `InnerException` parameter is niet <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, de huidige uitzondering is opgetreden tijdens een <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blok die verantwoordelijk is voor de interne uitzondering."
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.ArgumentNullException
  isExternal: true
  name: System.ArgumentNullException
- uid: System.Runtime.Remoting.RemotingException.#ctor
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException()
  nameWithType: RemotingException.RemotingException()
  fullName: System.Runtime.Remoting.RemotingException.RemotingException()
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String)
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException(String)
  nameWithType: RemotingException.RemotingException(String)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException(SerializationInfo,StreamingContext)
  nameWithType: RemotingException.RemotingException(SerializationInfo,StreamingContext)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String,System.Exception)
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException(String,Exception)
  nameWithType: RemotingException.RemotingException(String,Exception)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Runtime.Remoting.RemotingException.#ctor*
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException
  nameWithType: RemotingException.RemotingException
